scripts/scr_chapter_new/scr_chapter_new.gml

```diff
@@ -598,7 +651,7 @@ if (argument0="Crimson Fists"){founding="N/A";points=150;
 	//generates custom chapter if it exists
 	if (argument0=chapter21){
 		points=100;
-	    selected_chapter=21;chapter=argument0;icon=icon21;icon_name=icon_name21;founding=founding21;
+	    selected_chapter=22;chapter=argument0;icon=icon21;icon_name=icon_name21;founding=founding21;
 	    fleet_type=fleet_type21;
 		strength=strength21;
 		purity=purity21;
```

Comment:
could there be any errors here? as I see you've replaced chapter with 22, but I see a bunch of 21 here.
that's what I was talking about in, regards to custom chapters.